authn/api/http/endpoint_test.go

```diff
@@ -219,7 +219,7 @@ func TestRetrieve(t *testing.T) {
 			desc:   "retrieve a key unauthorized",
```

Comment:
If you have time I would appreciate that you also fix this logs. It should be `retrieve a token unauthorized`

Reply:
Ok, I'll do